nautobot/extras/models/groups.py

```diff
@@ -147,19 +148,19 @@ def _map_filter_fields(self):
         """Return all FilterForm fields in a dictionary."""
 
         # Fail gracefully with an empty dict if nothing is working yet.
-        if not self.form_class:
+        if self.form_class is None or self.filterform_class is None or self.filterset_class is None:
```

Comment:
I tried to add enough type annotations and checks here that `not-callable` wouldn't trip below, but ultimately gave up.